### ISSUE TITLE
Transfer apps to different node

### ISSUE THREAD

**Issue by docimin** [Dokploy/dokploy#812]
### What problem will this feature address?

Currently, as far as I know, you cannot transfer applications, database, compose apps etc. to a different node. This means you need to create a new one on the correct server and then copy over the files manually.. or more, depending ln the app.

### Describe the solution you'd like

Add some sort of "Transfer" in the advanced tab, where you can transfer an app to a different node.

### Describe alternatives you've considered

Copying files over, which is a lot of work.

### Will you send a PR to implement it?

No

**Comment by kerimovok** [comment]
related https://github.com/Dokploy/dokploy/issues/709#issue-2664978141 ?

**Comment by docimin** [comment]
> related [#709 (comment)](https://github.com/Dokploy/dokploy/issues/709#issue-2664978141) ?

I wouldn't say it's the same thing. The said issue is about backing up dokploy itself. My issue is about being able to transfer single applications/docker compose/databases to a different node, which is not the same as backup-ing the entire Dokploy instance itself.

**Comment by ipaderi** [comment]
Up, nice feature!

**Comment by Ayfri** [comment]
Any update ?

**Comment by mstrYoda** [comment]
What is the current state of this issue?

Is this planned or will accept a pr for this?

**Comment by jmlaya** [comment]
**As a workaround, you can update the Dokploy Postgres database directly.**

1. Connect to the server via SSH
`$ ssh usuario@ip-del-servidor`
2. List containers
`$ docker ps`
_Example output:_
```
CONTAINER ID   IMAGE          COMMAND              NAMES
b8f7e0d19d8f   nginx:latest   "/docker-entrypoint…"   dokploy-postgres....
```
3. Connect to the container using the container ID
`$ docker exec -it b8f7e0d19d8f /bin/sh`
_Tip: If that doesn't work, try `/bin/bash` instead._
4. Access Postgres inside the container:
`$ psql -U dokploy`
5. Use SQL to update the information. Table names are fairly intuitive — for example, to view organizations and projects data, use the `organization` and `project` tables. At this point, you can run `UPDATE` statements to modify fields like `organizationId` in project table, or `projectId` in application, mysql or postgres tables.
---
**NOTE:** 
Within the PSQL terminal, you can use the following commands for exploration:
- `\d` – list all tables
- `\d [table_name]` – list fields and indexes of a table
- `\l` – list databases
---
> ⚠️ **Disclaimer:**  
> Directly modifying the database can lead to data inconsistency, corruption, or loss if not done carefully.  
> Make sure you understand the structure and relationships of the tables before running any `UPDATE`, `DELETE`, or `INSERT` statements.  
> It is strongly recommended to back up the database first. Proceed with caution.


Cheers!.

**Comment by debojyoti452** [comment]
> **As a workaround, you can update the Dokploy Postgres database directly.**
> 
> 1. Connect to the server via SSH
>    `$ ssh usuario@ip-del-servidor`
> 2. List containers
>    `$ docker ps`
>    _Example output:_
> 
> ```
> CONTAINER ID   IMAGE          COMMAND              NAMES
> b8f7e0d19d8f   nginx:latest   "/docker-entrypoint…"   dokploy-postgres....
> ```
> 
> 3. Connect to the container using the container ID
>    `$ docker exec -it b8f7e0d19d8f /bin/sh`
>    _Tip: If that doesn't work, try `/bin/bash` instead._
> 4. Access Postgres inside the container:
>    `$ psql -U dokploy`
> 5. Use SQL to update the information. Table names are fairly intuitive — for example, to view organizations and projects data, use the `organization` and `project` tables. At this point, you can run `UPDATE` statements to modify fields like `organizationId` in project table, or `projectId` in application, mysql or postgres tables.
> 
> **NOTE:** Within the PSQL terminal, you can use the following commands for exploration:
> 
> * `\d` – list all tables
> * `\d [table_name]` – list fields and indexes of a table
> * `\l` – list databases
> 
> > ⚠️ **Disclaimer:**
> > Directly modifying the database can lead to data inconsistency, corruption, or loss if not done carefully.
> > Make sure you understand the structure and relationships of the tables before running any `UPDATE`, `DELETE`, or `INSERT` statements.
> > It is strongly recommended to back up the database first. Proceed with caution.
> 
> Cheers!.

Yes I can confirm it worked for me. But yes backup is must before proceeding!

**Comment by shreekrishnalamichhane** [comment]
If there is a valid solution with directly updating the database, why not add a native support in the UI ?

Also, I may not seeing in correct place, I can choose remote server while creating the service but after deploying to the remote server, I am not seeing that info anywhere. So with the server having very large number of services, it creates confusion on where the app is, unless you go and check the "Show Docker Containers" in the individual server on remote servers list.

**Comment by JoshuaRileyDev** [comment]
Is this ever going to be implemented? What's the reason why it won't work?

**Comment by mhbdev** [comment]
Is this planned? It is so hard to move our apps and services one by one and setup them from scratch in another server.

**Comment by JoshuaRileyDev** [comment]
I ended up adding it into my own self hosted fork

**Comment by mhbdev** [comment]
@JoshuaRileyDev Is the fork public?

**Comment by watzon** [comment]
Think you could PR it @JoshuaRileyDev? Even if it's not in a perfect state.

**Comment by JoshuaRileyDev** [comment]
https://github.com/JoshuaRileyDev/dokploy

It also has one click server provisioning via Hetzner, I love that feature the most as I use it all the time

**Comment by watzon** [comment]
Please PR both of those if you could. Both would be game changers for me, and I'm sure plenty of others.

**Comment by sandriaas** [comment]
any updates?

**Comment by NeoMetric** [comment]
This feature is a must. Please add it.

**Comment by JoshuaRileyDev** [comment]
I have this on my own fork which works really well

**Comment by wemiprog** [comment]
Any news on this?

**Comment by narcisonunez** [comment]
I'm looking into this. @JoshuaRileyDev a PR is welcome :)

**Comment by JoshuaRileyDev** [comment]
Oh ok, I’ll see how easy it is to extract from my fork, I have a few other
features in mine like auto provisioning Hetzner servers etc so need to
cherry pick.

**Comment by Siumauricio** [comment]
Hey Guys, this feature will be included in our next release!
